Assets/__Scripts/UI/FirstBootMenu.cs

```diff
@@ -61,4 +68,25 @@ public class FirstBootMenu : MonoBehaviour {
         helpPanel.SetActive(!helpPanel.activeSelf);
     }
 
+
+    private string guessBeatSaberInstallationDirectory()
+    {
+        //TODO: find out potentiell Oculus Store installation location
```

Comment:
You can borrow code from [ModAssistant](https://github.com/Assistant/ModAssistant/blob/master/ModAssistant/Classes/Utils.cs#L253) to find the install location for the Oculus Store version.